src/components/error-page/useIsMobile.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import { mobileScreenMaxWidth } from "./styles";
+
+const mediaQueryMobile = window.matchMedia(`(max-width: ${mobileScreenMaxWidth}px)`);
+
+const getisMobile = (): boolean => {
+  return mediaQueryMobile.matches;
+};
+
+export const useIsMobile = () => {
+  const [isMobile, setIsMobile] = useState<boolean>(getisMobile());
+
+  useEffect(() => {
+    const resize = () => {
+      setIsMobile(getisMobile());
```

Comment:
should check before call because i don't sure that setMethods for react check something here or just would be rerender on every resizing event